python/tests/test_coworker.py

```
@@ -19,6 +19,25 @@ def test_persistent_and_alias_delta_default_tier(self):
+            assert mem.table_name == "main.default.apx_memories"
+            assert sess.table_name == "main.default.apx_sessions"
+
+    def test_persistent_with_catalog_derives_uc_table_names(self):
+        mem, sess = normalize_memory_knob("persistent", catalog="acme", schema="hr", name="hr_coworker")
+        assert mem.type == "delta"
+        assert mem.table_name == "acme.hr.apx_hr_coworker_memory"
+        assert sess.table_name == "acme.hr.apx_hr_coworker_sessions"
+
+    def test_persistent_with_catalog_slugifies_name(self):
+        # Non-alphanumeric chars → underscores; trailing stripped
+        mem, sess = normalize_memory_knob("persistent", catalog="cat", schema="sch", name="my-agent name!")
```

Ruff (0.15.15)

[warning] Unpacked variable `sess` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)